File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/AllowanceOutputDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[FunctionOutput]>
    Public Class AllowanceOutputDTO
    
        <[Parameter]("uint256", "remaining", 1)>
        Public Property Remaining As BigInteger
    
    End Class

End Namespace

File: src/Nethereum.StandardTokenEIP20/DTOs/AllowanceOutputDTO.cs
using System.Numerics;
using Nethereum.ABI.FunctionEncoding.Attributes;
using Nethereum.Contracts;

namespace Nethereum.StandardTokenEIP20.DTOs
{
    [FunctionOutput]
    public class AllowanceOutputDTO:IFunctionOutputDTO
    {
        [Parameter("uint256", "remaining", 1)]
        public BigInteger Remaining {get; set;}
    }
}
